src/main/java/com/opencart/pages/containers/ShoppingCartProductContainer.java

```diff
@@ -0,0 +1,16 @@
+package com.opencart.pages.containers;
+
+import com.opencart.pages.BasePage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class ShoppingCartProductContainer extends BasePage {
+
+    public ShoppingCartProductContainer() {
+    }
+
+    public WebElement getTotalPrice(String productName) {
+        return driver.findElement(By.xpath(String.format("//*[@class='table table-bordered']//*[text()='%s']/ancestor::tr//*[@class='text-right'][last()]", productName)));
```

Comment:
I am not sure if you need this part of xpath here - "*[@class='table table-bordered']//"

Reply:
If l don't use this part of the XPath it gets to find more than one locator